tensorflow/contrib/gdr/gdr_rendezvous_mgr.cc

```diff
@@ -170,6 +170,15 @@ class GdrRemoteRendezvous : public BaseRemoteRendezvous {
     // Record "call" in active_ so that it can be aborted cleanly.
     RegisterCall(call);
 
+    // RendezvousMgr already aborted, shouldn't send RPC call any more
+    if (!call->status().ok()) {
+      call->done()(call->status(), Args(), Args(), Tensor(), false);
```

Comment:
Does `GdrRecvTensorCall` have a `done()` method? Its constructor doesn't seem to take `done` as an argument, so I'm not sure what's getting called here (or if this even compiles!).

Reply:
U r right, merge code mistake here. Fixed,  thx mrry.